tools/run_tests/xds_k8s_test_driver/framework/xds_url_map_testcase.py

```diff
@@ -102,7 +102,7 @@ def __init__(self, xds_json: JsonType):
                     for endpoint in xds_config['endpointConfig'][
                             'dynamicEndpointConfigs']:
                         self.eds.append(endpoint['endpointConfig'])
-            except Exception as e:
+            except Exception as e:  # pylint: disable=broad-except
```

Comment:
Here and below: 
Do we really need catching `Exception` here? Looking at the code, it can be replaced with `LookupError`. But I may be wrong.